### DISCUSSION TITLE
Issue of flags -govs-include -govs-moltype in martinize2

### DISCUSSION THREAD

**Discussion by avinash031997** [Martini-Force-Field-Initiative/MartiniDiscussions#23] [General]
I am setting up a protein with Go-like model, initially, I am using martinize2 command 

martinize2 -f aa.pdb -o system.top -x CG.pdb -dssp  -p backbone -ff martini3001 -govs-include -govs-moltype PS -scfix -cys auto

then  am getting an error:

martinize2: error: unrecognized arguments: -govs-include -govs-moltype t4l
 
 Have these options been removed or should I use other references?
 
 Please give me suggestions on this issue.
 
 Thank you in advance.

**Comment by csbrasnett** [comment]
Just fyi in future this may be better placed in the [Vermouth discussions board](https://github.com/marrink-lab/vermouth-martinize/discussions)

These arguments have been removed from the latest versions of Martinize2, because they're no longer needed with the current implementation of the Go model. I've written a small tutorial on how it should be done [here](https://vermouth-martinize.readthedocs.io/en/latest/tutorials/go_models.html), the one on the website needs updating asap

**Comment by avinash031997** [comment]
Thank you so much for replying to my query.

I am doing according to this tutorial but I do not understand the issue. I am attaching the error:
Traceback (most recent call last):
  File "/home/user/.local/bin/martinize2", line 1153, in <module>
    entry()
       return self.__chain_id_to_resnode[(chain, resid)]
KeyError: ('Z', 1)

Please help me regarding this issue.

Thank you in advance.

**Reply by csbrasnett** [reply]
I've usually seen this issue when the pdb you use to generate the contact map doesn't have chains written. In this case, the contact map server will add the chain indicator "Z" to the contact map file, which is then not found by martinize2. You should be able to solve this by rewriting the chain into the input pdb, regenerating the contact map, and the running martinize2 on the protein.

**Comment by avinash031997** [comment]
It's functioning correctly now. Thank you very much, Chris Brasnett, for your assistance with these issues.
